bridge/device/cloud/manager.go

```diff
@@ -347,11 +361,15 @@ func validUntil(expiresIn int64) time.Time {
 }
 
 func (c *Manager) setCreds(creds ocfCloud.CoapSignUpResponse) {
+	c.private.mutex.Lock()
+	defer c.private.mutex.Unlock()
 	c.creds = creds
 	c.signedIn = false
 }
 
 func (c *Manager) getCreds() ocfCloud.CoapSignUpResponse {
+	c.private.mutex.Lock()
+	defer c.private.mutex.Unlock()
 	return c.creds
```

Comment:
I'd move `c.creds` to `c.private.creds`, since members protected by the mutex should be in the `private` struct

Reply:
fixed